test/Panama/src/org/openj9/test/panama/FunctionPointerTest.java

```diff
@@ -0,0 +1,198 @@
+/*******************************************************************************
+ * Copyright (c) 2012, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+package org.openj9.test.panama;
+
+import org.testng.annotations.*;
+import org.testng.Assert;
+
+import java.lang.invoke.*;
+import java.nicl.*;
+
+import panama.test.panamatest;
+
+public class FunctionPointerTest {
+	panamatest panama;
+
+	@Parameters({ "path" })
+	public void setUp(String path) throws Exception {
+		Library lib = NativeLibrary.loadLibraryFile(path + "/libpanamatest.so");
+		panama = NativeLibrary.bindRaw(panamatest.class, lib);
+	}
+	
+	@Test
+	public void testVoidVirtual() throws Throwable {
```

Comment:
We do not need to annotate on each method as long as we annotate at class level.